doc/layouts/project/list.html

```diff
@@ -1,3 +1,4 @@
+{{ define "main" }}
 <!--
```

Comment:
I didn't notice earlier, this needs to be inside the `{{ define "main" }}` .  I'm not sure why.